docs/introduction.rst

```diff
@@ -1,73 +1,57 @@
+********
+Nion Swift is an image processing software for controlling instruments, and managing the acquisition, organization, visualization, and analysis of data. The software was designed to be used in conjunction with Nion's electron microscopes, but the functionality and the ability to extend its capabilities using Python mean that Nion Swift is useful for any number of data visualization projects. Whether you already have your data, or you want to see the data as it is acquired, Nion Swift helps you visualize your data in exactly the way you want.
 
-|AppName| is intended to provide an extensible platform to acquire scientific data from a variety of microscopes and other instruments, be intuitive and easy to use, and allow sophisticated data visualization, processing, and analysis.
+Click to download Nion Swift or for :ref:`installation` instructions.
 
-Key Features
+Click here to get started fast with our :ref:`basic-use`.
 
-- Data handling of 1D plots, 2D images, 1D and 2D collections of plots and images, and sequences.
-- Live computations that can be chained during acquisition or live parameter adjustment.
-- Open source, cross platform (macOS, Windows, Linux), and Python based scientific data processing.
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is split up into a central section of the window called the workspace and the surrounding area for utility panels. 
 
-Data Items and Display Items
-----------------------------
-|AppName| operates on data arrays, which include 2D images and 1D spectra, which are generally called *Data Items*.
+The central workspace is for display panels which can either display a single piece of data (also called a data item) or can display multiple pieces of data side by side in one of the browser formats. For more information, see :ref:`display-panels`.
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The utility panels surrounding the central workspace allow you to make adjustments to a selected data item in a display panel. For more information on their specific functions, see :ref:`user-interface`.
 
-A spectrum image would be a 2D collection of 1D data. This would be typical data acquired from a STEM microscope spectrum imaging acquisition.
+Data and Display
+================
+A piece of data stored in the program is called a data item. Data items can be dragged into display panels to be displayed. Once displayed, they become display items. Display items can be modified and viewed to see various aspects or values of the original data item. Read more in :ref:`data-items`.
```

Comment:
Again, I'd avoid "piece of data" since it is just a renaming of "item of data" and just use something simpler like "Data stored in the application is called..." or "Data items stored in ...".

Also, I'd like to stick with "application" rather than "program"; but ideally avoid those altogether.

"they become display items" -- well, not really: they _are_ display items. "become" implies some sort of action.